chromium_src/components/sync/base/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+  "+brave/components/brave_sync",
```

Comment:
pls make this explicit: `"+brave/components/brave_sync/buildflags.h",`, because we can't include any file from `brave_sync/*` dir, but only those files that are wired via GN correctly.

Right now there's no GN dependency between `//components/sync/base/sync_util.h`  and `brave_sync:buildflags`, so please add `//brave/components/brave_sync:buildflags` as a `public_deps` into a target that includes `//components/sync/base/sync_util.h`.

Reply:
Done, Thanks the for details!